dawn/main/ansible/Ansible.js

```diff
@@ -16,12 +19,31 @@ const StudentCodeStatus = DawnData.StudentCodeStatus;
 
 const runtimeBuilder = ProtoBuf.loadProtoFile(`${protoFolder}/runtime.proto`);
 const RuntimeData = runtimeBuilder.build('RuntimeData');
+const TCPData = runtimeBuilder.build('TCPData');
+
+let test = 0;
```

Comment:
Actually, more reading shows this wont work: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let
`At the top level of programs and functions, let, unlike var, does not create a property on the global object.`